_posts/2023-08-22-using-plurimath.adoc

```diff
@@ -0,0 +1,282 @@
+---
+layout: post
+title:  "Using Plurimath"
+date:   2023-08-22 00:00:00 +0800
+categories:
+  - plurimath
+  - using
+authors:
+  -
+    name: Suleman Uzair
+    email: sulemanuzair600@gmail.com
+    social_links:
+      - https://github.com/suleman-uzair/
+  -
+    name: Ronald Tse
+    email: tse@ribose.com
+    use_picture: assets
+    social_links:
+      - https://www.linkedin.com/in/rhtse/
+      - https://github.com/ronaldtse
+excerpt: >-
+  Conversions across multiple representational languages is a core feature of Plurimath.
+  To make it even more accessible, you can easily perform conversions by specifying the input string's type.
+---
+
+= Using Plurimath
+
+== What is Plurimath?
+
+*Plurimath* is a comprehensive and efficient tool that enhances the way mathematical content is converted between different formats.
+It is designed to work with various markup languages such as *MathML*, *LaTeX*, *AsciiMath*, and *OMML*, making it a versatile and adaptable tool.
+
+One significant advantage is that users don't have to deal with the inconvenience of installing and managing multiple libraries for these different languages. *Plurimath* encompasses them all within a single solution.
+
+This approach simplifies and improves the process of converting mathematical content between markup languages, making it easier and more reliable.
+
+== 1. How to use Plurimath?
+
+Using *Plurimath* is straightforward and user-friendly, whether you're working on a *Ruby* application, a *Ruby on Rails* project, or any other framework based on *Ruby*.
+To get started, add this line into your applications gemfile.
+add this line into your applications gemfile
+
+```ruby
+  gem "plurimath"
+```
+then run following command
+```ruby
+  bundle install
+```
+or run following command
+```ruby
+  gem install plurimath
+```
+
+Once it's incorporated, you can seamlessly convert mathematical expressions between various markup languages.
+
+=== 1.1 Understanding Plurimath
+
+*Plurimath* provides a range of conversion methods, allowing you to specify the input type and obtain the desired output format.
+With just a few lines of code, you can harness the power of *Plurimath* to ensure consistency and compatibility in mathematical presentations across different platforms and applications.
+
+Now let's discuss *how to use Plurimath?*.
+
+```ruby
+  formula = Plurimath::Math.parse(string, :type)
+```
+
+You will have to provide input text replacing `string`, and the `type` field should specify the type of the provided text, choosing from one of the supported languages.
+
+The above example will generate a Formula object, something like this
+
+```ruby
+  Plurimath::Math::Formula.new([
+    ... # Others objects
+  ])
+```
+
+Now next step would be conversion(since you parsed your string into *Plurimath Formula*) to your desired output language. For conversion all you have to do is call method name containing language's name with leading `to_`.
```

Comment:
```suggestion
Now the next step would be conversion (since you have parsed your string into a *Plurimath Formula*) to your desired output format. For conversion all you have to do is call the method name containing the format name with a leading `to_`.
```